src/Microsoft.OData.Client/Materialization/ODataEntriesEntityMaterializer.cs

```diff
@@ -86,7 +86,7 @@ internal override long CountValue
         {
             get
             {
-                throw new InvalidOperationException(DSClient.Strings.MaterializeFromAtom_CountNotPresent);
+                throw new InvalidOperationException(DSClient.Strings.MaterializeFromObject_CountNotPresent);

```

Comment:
nit: Maybe "MaterializeFromEntry" better than "MaterializeFromObject" given that you're dropping "atom" from "atom entry" in other places...